osu.Game/Graphics/UserInterface/SliderBoxes.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public partial class SliderBoxes : Container
```

Comment:
Honestly, I'd just implement this locally twice and do away with this class? It's doing so little I don't think it's worthwhile.

Reply:
agreed